app/services/feature_flag_service/store.rb

```diff
@@ -19,6 +19,7 @@ class FeatureFlag
     FLAGS = [
       :export_transactions_as_csv,
       :topbar_v1,
+      :discovery_api
```

Comment:
This flag could be named to 'searchpage_v1' or something else since we are not going to be able to only turn discovery_api _on_. I.e. There's nothing that would be able to handle/render the results separately.